qemu/tests/cfg/blockdev_mirror_cancel_ready_job_with_ioerror.cfg

```diff
@@ -0,0 +1,23 @@
+- blockdev_mirror_cancel_ready_job_with_ioerror:
+    only Linux
+    only filesystem
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_mirror_cancel_ready_job_with_ioerror
+    virt_test_type = qemu
+    source_images = source
+    target_images = target
+    image_name_source = source
+    image_name_target = target
+    image_format_source = qcow2
+    image_format_target = qcow2
+    image_size_target = 128M
+    image_size_source = 128M
+    filter_node_name = mirror_top
+    backup_options_source = sync filter-node-name on-target-error
+    sync = full
+    filter_node_name = mirror_top
+    on-target-error = stop
+    storage_pools = default
+    storage_pool = default
+    storage_type_default = directory
```

Comment:
These are not needed, because we also use VT's storage module to handle both source and target

Reply:
@zhencliu, have removed it, thanks.